### ISSUE TITLE
How to set the hyperparameters

### ISSUE THREAD

**Issue by Qiuben** [orrzohar/PROB#27]
Hi @orrzohar 
Thanks for your nice works and share! Currently I want to reproduce your code with 2 3090 GPUs, I wonder how should I set the hyperparameters, which of them need to be changed essentially?

**Comment by orrzohar** [comment]
Hi @Qiuben,

I would start with the same hyper parameters reported in OW-DETR:

    parser.add_argument('--lr', default=2e-4, type=float)
    parser.add_argument('--lr_backbone', default=2e-5, type=float)
    parser.add_argument('--weight_decay', default=1e-4, type=float)
    parser.add_argument('--lr_drop', default=40, type=int)
    parser.add_argument('--clip_max_norm', default=0.1, type=float,
                        help='gradient clipping max norm')
    parser.add_argument('--mask_loss_coef', default=1, type=float)
    parser.add_argument('--dice_loss_coef', default=1, type=float)
    parser.add_argument('--cls_loss_coef', default=2, type=float)
    
    parser.add_argument('--bbox_loss_coef', default=5, type=float)
    parser.add_argument('--giou_loss_coef', default=2, type=float)
    parser.add_argument('--focal_alpha', default=0.25, type=float)

They trained on a system with 8 V100 GPUs, but the batch size there was 2. Do you know what batch size fits on your system?
Have you ever trained a D-DETR or OW-DETR model on your system? if so, what hyper parameters did you use?

Best,
Orr

**Comment by Qiuben** [comment]
Hi @orrzohar 
I'm sorry that I can't do the experimental verification in time now, because I have gone home for vacation. If I'm free later, I'll come back to this. Now I will close this issue!
